benchmark/wildtime_benchmarks/example_pipelines/yearbook.yaml

```diff
@@ -0,0 +1,50 @@
+pipeline:
+  name: Yearbook Test Pipeline
+  description: Example pipeline
+  version: 1.0.0
+model:
+  id: YearbookNet
+  config:
+    num_input_channels: 1
+    num_classes: 2
+training:
+  gpus: 1
+  device: "cpu"
+  dataloader_workers: 2
+  use_previous_model: True
+  initial_model: random
+  initial_pass:
+    activated: False
+  batch_size: 64
+  optimizers:
+    - name: "default"
+      algorithm: "SGD"
+      source: "PyTorch"
+      param_groups:
+        - module: "model"
+          config:
+            lr: 0.001
+            momentum: 0.9
+  optimization_criterion:
+    name: "CrossEntropyLoss"
+  checkpointing:
+    activated: False
+  selection_strategy:
+    name: NewDataStrategy
+    maximum_keys_in_memory: 1000
+    config:
+      limit: -1
+      reset_after_trigger: True
+data:
+  dataset_id: yearbook
+  transformations: []
+  bytes_parser_function: |
+    import torch
+    import numpy as np
+    def bytes_parser_function(data: bytes) -> torch.Tensor:
+      return torch.from_numpy(np.frombuffer(data, dtype=np.float32)).reshape(1, 32, 32)
```

Comment:
any reason that we do it like that and not like fmow?

Reply:
ok even if we use the binary file wrapper here, do we not need any kind of image transformations, like Normalize? I thought normalizing the input as with MNIST or the FMOW data was pretty standard

Reply:
I thought so too, but based on WildTimeData it seems that the only transformation is to create the tensor

Reply:
See [Here](https://github.com/wistuba/Wild-Time-Data/blob/main/wild_time_data/datasets.py) line 187 vs line 163 (FMoW)

Reply:
Hmm, ok, then I suppose it is correct that way.